### ISSUE TITLE
[A11yInspect] Fail: 1.3.1 A in /home/runner/work/test-app/test-app/src/sample.jsx

### ISSUE THREAD

**Issue by sitaram-r** [sitaram-r/test-app#10]
## ❌ Accessibility Fail

**Rule:** 1.3.1 A
**File:** `/home/runner/work/test-app/test-app/src/sample.jsx`
**Line:** 13, **Column:** 1

### Description
Empty heading (Minor)

### Context
```html
<h4>Skipped Heading</h4>
```

### Selector
`html > body > div > h4`

---
*Generated by [A11yInspect](https://github.com/barrierbreak/vscode-scally)*
